saleor/webhook/event_types.py

```diff
@@ -18,6 +19,11 @@ class WebhookEventAsyncType:
     CATEGORY_UPDATED = "category_updated"
     CATEGORY_DELETED = "category_deleted"
 
+    GIFT_CARD_CREATED = "gift_card_created"
+    GIFT_CARD_UPDATED = "gift_card_updated"
+    GIFT_CARD_DELETED = "gift_card_deleted"
+    GIFT_CARD_STATUS_CHANGED = "gift_status_changed"
```

Comment:
```suggestion
    GIFT_CARD_STATUS_CHANGED = "gift_card_status_changed"
```
We should re-generate the schema after this change I guess.